docs/guides/monitor-job.mdx

```diff
@@ -50,6 +50,64 @@ You can cancel a job from the IBM Quantum Platform dashboard either on the Jobs
 
 In Qiskit, use `job.cancel()` to cancel a job.
 
+<span id="execution-spans"></span>
+## View Sampler execution spans
+
+The results of `SamplerV2` jobs executed in Qiskit Runtime contain execution timing information in their metadata.
```

Comment:
Out of curiosity, is there some kind of magic that goes through these markdowns and convert class names to API links?

Reply:
No; I need to add links if we want them.

Reply:
I added a link to the first reference.